warehouse/query-core/src/main/java/datawave/query/tables/ScannerFactory.java

```diff
@@ -282,7 +322,7 @@ public void setMaxQueue(int size) {
     public synchronized ScannerBase newRfileScanner(String tableName, Set<Authorizations> auths, Query setting) {
         Configuration conf = new Configuration();
 
-        AccumuloClient con = cxn;
+        AccumuloClient con = client;
```

Comment:
Can `con` just be dropped and we use `client` throughout this method?

Reply:
We are trying to be colorful in our use of variables.  Its a con artist.

Reply:
Removed in a later commit